source/content/guides/drupal/drupal-hosted-createbt/05-upgrade.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: Migrate a Site That Was Created with Build Tools to Drupal 9
```

Comment:
@stovak references `d9` in code samples.  see preview https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-hosted-createbt/upgrade